scss/style.scss

```diff
@@ -1,14 +1,8 @@
 // ==========================================================================
 // CSS Chassis
 // ==========================================================================
+// This just add normalize to the build see lint.scss for expaination.
 
 @import
-	"_utilities/clearfix",
-	"_utilities/colors";
-
-@import
-	"atoms/icons/icons",
-	"atoms/typography/typography";
-
-@import
-	"views/main";
+	"external/normalize.css/normalize.scss",
+	"lint.scss";
```

Comment:
It needs to be compiled in 2 locations that the whole point we need to make a compiled file to lint.